### PR TITLE
Stream live data

### PR DESCRIPTION
**Pull request by rchaing-figure**
Note: This PR is the initial implementation of handling historical and live streaming of Coin Transfer, Marker Supply and Marker Transfer the coding is less efficient as hope so that we can debug the stream while being hammered from the client side (TA);

### CODE REVIEW

File: src/main/kotlin/tech/figure/aggregate/service/stream/consumers/EventStreamUploader.kt [hunk 1]
```diff
@@ -182,7 +189,13 @@ class EventStreamUploader(
                                         // Handle inserting data into postgres.
                                         // todo: want to return a proper status.
                                         transaction {
-                                            dbClient.handleInsert(extractor.name, out.path.toFile())
+                                            dbClient.handleInsert(
+                                                extractor.name,
+                                                out.path.toFile(),
+                                                coinTransferChannel,
+                                                markerSupplyChannel,
+                                                markerTransferChannel
```
Comment (rchaing-figure) [inline src/main/kotlin/tech/figure/aggregate/service/stream/consumers/EventStreamUploader.kt:197]: will fix, very ugly

File: src/test/kotlin/tech/figure/aggregate/service/EventStreamUploaderTests.kt [hunk 2]
```diff
@@ -34,6 +37,10 @@ class EventStreamUploaderTests {
     val log: Logger = logger()
     val ravenClient = mockk<RavenDB>()
 
+    val channelCoinTransfer = mockk<Channel<CoinTransfer>>()
+    val channelMarkerSupply = mockk<Channel<MarkerSupply>>()
+    val channelMarkerTransfer = mockk<Channel<MarkerTransfer>>()
```
Comment (rchaing-figure) [inline src/test/kotlin/tech/figure/aggregate/service/EventStreamUploaderTests.kt:42]: will fix, very ugly

File: src/test/kotlin/tech/figure/aggregate/service/EventStreamUploaderTests.kt [hunk 3]
```diff
@@ -81,6 +88,9 @@ class EventStreamUploaderTests {
                     uploadResults1 = EventStreamUploader(
                         blockFlow,
                         dbClient,
+                        channelCoinTransfer,
+                        channelMarkerSupply,
+                        channelMarkerTransfer,
```
Comment (rchaing-figure) [inline src/test/kotlin/tech/figure/aggregate/service/EventStreamUploaderTests.kt:93]: will fix, very ugly

File: src/main/kotlin/tech/figure/aggregate/service/Main.kt [hunk 4]
```diff
@@ -249,6 +258,9 @@ fun main(args: Array<String>) {
         EventStreamUploader(
             blockFlow,
             dbClient,
+            coinTransferChannel,
+            markerSupplyChannel,
+            markerTransferChannel,
```
Comment (rchaing-figure) [inline src/main/kotlin/tech/figure/aggregate/service/Main.kt:263]: will fix, very ugly

**Review by hyperschwartz — APPROVED** [review]

**Pull request merged**